BitwardenShared/UI/Platform/Application/Views/BitwardenTextField.swift

```diff
@@ -32,6 +32,9 @@ struct BitwardenTextField<TrailingContent: View>: View {
     /// Whether a password in this text field is visible.
     let isPasswordVisible: Binding<Bool>?
 
+    /// If the keyboard should be presented immediately when the view appears.
+    let isPasswordAutoFocused: Bool
```

Comment:
🤔 Is it possible to use the `focused` modifier to set focus on `BitwardenTextField` in the containing view without introducing changes to `BitwardenTextField`? I think we've done that elsewhere and that feels a little bit more SwiftUI-y.

Reply:
just for historical purposes, this wasn't working for the `pin` section because of how the views are laid out.  Switching back to the original implementation which worked for both.